### ISSUE TITLE
Topic Idea: How to write image alt-text

### ISSUE THREAD

**Issue by courtneyr-dev** [WordPress/Learn#1463]
# Details

- Topic title: How to write image alt-text
- Topic description: Primarily focus on what creates quality alt text descriptions, but demonstrate how to add these in WordPress. This should be a brief lesson plan and tutorial video solely focused on this, but can be referred to from any other media related content and from the Photos team.
- Objective: The objective of this lesson is to teach how to write effective alt-text for images and apply it in WordPress for better accessibility. Students will learn to identify key components of good alt-text, write alt-text, and implement it in WordPress.
- Content type: lesson plan, tutorial
- WordPress version: current
- Will you be creating this content?: No

# Related Resources
Links to related content on Learn, HelpHub, DevHub, GitHub Gutenberg Issues, DevNotes, etc.
- [ ] [Slack log](https://wordpress.slack.com/archives/C02QB8GMM/p1680734099055619?thread_ts=1680730665.281389&cid=C02QB8GMM)

# Triage Checklist
Please follow this [Vetting content topic ideas handbook page]() guide.

- [ ] SME assigned
- [ ] Remove Needs Subject Matter Expert label
- [ ] Content Category applied (Ex. PHP, Gutenberg, Block Theme)
- [ ] Topic idea has passed vetting checks
- [ ] Content type label applied
- [ ] Priority set
- [ ] Content Type Checklist applied
- [ ] Content type
- [ ] Audience set
- [ ] Experience Level set
- [ ] Product type (theme, plugin)
- [ ] WordPress Version applied through Milestone (Ex. milestone 5.9)
- [ ] If the Topic Idea reporter has not opted to create the content, apply Needs Content Creator label
- [ ] Ticket moved to Ready to Create

**Comment by wpbokul** [comment]
I will be happy to make an online workshop on this topic. @devmuhib009
